mlperf_logging/compliance_checker/mlp_compliance.py

```diff
@@ -41,7 +41,8 @@ def merge(*dicts,):
 
 class ComplianceChecker:
 
-    def __init__(self, ruleset, quiet, werror):
+    def __init__(self, usage, ruleset, quiet, werror):
```

Comment:
Have you tried to run standalone MLP compliance for a single log file? I tried and it fails, if you have please provide the command line for hpc/training. Also probably you will need to change the compliance_checker/README.md dile

Reply:
I ran

`python3 -m mlperf_logging.package_checker <1.0.0 results folder> training 1.0.0 --rcp_bert_train_samples`

on the NVIDIA and Google folders and everything passed. For example for the Google result:

```SUCCESS
Table CSV prefix: Google,tpu-v4-256,AMD Rome,256,TPU,4,JAX,
Table CSV postfix: ,https://github.com/mlperf/training_results_v1.0.0/blob/master/Google/systems/tpu-v4-256.json,https://github.com/mlperf/training_results_v1.0.0/blob/master/Google/benchmarks,
SUCCESS
Table CSV prefix: Google,tpu-v4-4096,AMD Rome,256,TPU,4,TensorFlow,
Table CSV postfix: ,https://github.com/mlperf/training_results_v1.0.0/blob/master/Google/systems/tpu-v4-4096.json,https://github.com/mlperf/training_results_v1.0.0/blob/master/Google/benchmarks,
SUCCESS
Table CSV prefix: Google,tpu-v4-2048,AMD Rome,256,TPU,4,TensorFlow,
Table CSV postfix: ,https://github.com/mlperf/training_results_v1.0.0/blob/master/Google/systems/tpu-v4-2048.json,https://github.com/mlperf/training_results_v1.0.0/blob/master/Google/benchmarks,
SUCCESS
Table CSV prefix: Google,tpu-v4-256,AMD Rome,256,TPU,4,TensorFlow,
Table CSV postfix: ,https://github.com/mlperf/training_results_v1.0.0/blob/master/Google/systems/tpu-v4-256.json,https://github.com/mlperf/training_results_v1.0.0/blob/master/Google/benchmarks,
SUCCESS
Table CSV prefix: Google,tpu-v4-128,AMD Rome,256,TPU,4,TensorFlow,
Table CSV postfix: ,https://github.com/mlperf/training_results_v1.0.0/blob/master/Google/systems/tpu-v4-128.json,https://github.com/mlperf/training_results_v1.0.0/blob/master/Google/benchmarks,
SUCCESS
Table CSV prefix: Google,tpu-v4-1024,AMD Rome,256,TPU,4,JAX,
Table CSV postfix: ,https://github.com/mlperf/training_results_v1.0.0/blob/master/Google/systems/tpu-v4-1024.json,https://github.com/mlperf/training_results_v1.0.0/blob/master/Google/benchmarks,
SUCCESS
Table CSV prefix: Google,tpu-v4-6912,AMD Rome,256,TPU,4,TensorFlow,
Table CSV postfix: ,https://github.com/mlperf/training_results_v1.0.0/blob/master/Google/systems/tpu-v4-6912.json,https://github.com/mlperf/training_results_v1.0.0/blob/master/Google/benchmarks,
SUCCESS
Table CSV prefix: Google,tpu-v4-128,AMD Rome,256,TPU,4,JAX,
Table CSV postfix: ,https://github.com/mlperf/training_results_v1.0.0/blob/master/Google/systems/tpu-v4-128.json,https://github.com/mlperf/training_results_v1.0.0/blob/master/Google/benchmarks,```

Reply:
done, works